corehq/apps/users/models.py

```diff
@@ -1567,24 +1568,16 @@ def get_owner_ids():
         user._hq_user = self # don't tell anyone that we snuck this here
         return user
 
-    def get_forms(self, deleted=False, wrap=True, include_docs=False):
+    def get_forms(self, deleted=False, wrap=True):
+        accessor = FormAccessors(self.domain)
         if deleted:
-            view_name = 'deleted_data/deleted_forms_by_user'
-            startkey = [self.user_id]
+            doc_ids = accessor.get_deleted_form_ids_for_user(self.user_id)
         else:
-            view_name = 'all_forms/view'
-            startkey = ['submission user', self.domain, self.user_id]
+            doc_ids = accessor.get_form_ids_for_user(self.domain, self.user_id)
 
-        db = XFormInstance.get_db()
-        doc_ids = [r['id'] for r in db.view(view_name,
-            startkey=startkey,
-            endkey=startkey + [{}],
-            reduce=False,
-            include_docs=False,
-        )]
-        if wrap or include_docs:
-            for doc in iter_docs(db, doc_ids):
-                yield XFormInstance.wrap(doc) if wrap else doc
+        if wrap:
```

Comment:
:+1:, still need to add that logic. got frustrated with the test not showing the xform instance haha